DashKit/DashPumpManagerState.swift

```diff
@@ -186,8 +197,12 @@ public struct DashPumpManagerState: RawRepresentable, Equatable {
             "finishedDoses": finishedDoses.map( { $0.rawValue }),
             "basalProgram": basalProgram.rawValue,
             "maximumTempBasalRate": maximumTempBasalRate,
-            "activeAlerts": activeAlerts.rawValue
+            "activeAlerts": activeAlerts.rawValue,
         ]
+        
+        if let rawLastPodCommState = try? JSONEncoder().encode(lastPodCommState) {
```

Comment:
[nit] I believe this can be shortened to just `rawValue["lastPodCommState"] = try? JSONEncoder().encode(lastPodCommState)`.